Jenkinsfile.release

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env groovy
+
+ansiColor('xterm') {
+  node('JenkinsMarathonCI-Debian8-2017-10-23') {
+
+    properties([
+      parameters([
+        string(name: 'gitsha',
+          defaultValue: '',
+          description: 'Git Commit SHA to Build and Release (ex. 167d7fb86 or 167)'
+        ),
+        string(name: 'version',
+          defaultValue: '',
+          description: 'Community Release Version (ex. v1.5.2)'
+        ),
+        string(name: 'latest',
+          defaultValue: 'False',
+          description: 'True if docker mesosphere/marathon:latest should be pushed'
+        )]
+      )
+    ])
+
+    stage("Run Pipeline") {
+      try {
+        checkout scm
+        withCredentials([
+            usernamePassword(credentialsId: 'a7ac7f84-64ea-4483-8e66-bb204484e58f', passwordVariable: 'GIT_PASSWORD', usernameVariable: 'GIT_USER'),
+            string(credentialsId: '3f0dbb48-de33-431f-b91c-2366d2f0e1cf',variable: 'AWS_ACCESS_KEY_ID'),
+            string(credentialsId: 'f585ec9a-3c38-4f67-8bdb-79e5d4761937',variable: 'AWS_SECRET_ACCESS_KEY'),
+            string(credentialsId: '7bdd2775-2911-41ba-918f-59c8ae52326d',variable: 'DOCKER_HUB_USERNAME'),
+            string(credentialsId: '42f2e3fb-3f4f-47b2-a128-10ac6d0f6825',variable: 'DOCKER_HUB_PASSWORD'),
+            string(credentialsId: '4551c307-10ae-40f9-a0ac-f1bb44206b5b',variable: 'DOCKER_HUB_EMAIL'),
+        ]) {
+          sh """docker login -u ${DOCKER_HUB_USERNAME} -p ${DOCKER_HUB_PASSWORD} -e ${DOCKER_HUB_EMAIL}"""
```

Comment:
Checkout the [Jenkinsfile from marathon-dcos-plugins](https://github.com/mesosphere/marathon-dcos-plugins/blob/master/Jenkinsfile#L19). We used

```
withDockerRegistry([credentialsId: 'docker-hub-credentials', url: 'https://index.docker.io/v1/']) {
  sh """sudo -E ci/pipeline release $params.version $params.gitsha $params.latest"""
}
```

that seems a little cleaner to me.

Reply:
I agree!!  on it!